frontend/src/store/index.js

```diff
@@ -10,7 +16,7 @@ export function* signupRequest(user) {
 		yield put({type: actions.user.SIGNUP_FAILED})
 	}
 	else {
-		yield put({type: actions.user.SIGNUP, user: user})
```

Comment:
SIGNUP and SIGNUP_DONE is different action.
SIGNUP_DONE is always called when the user leave the signup_done page. Reverted

Reply:
[Edited] SIGNUP is not useful also. SIGNUP_SUCCESSFUL will be replace it.